runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/TestStreamTranslation.java

```diff
@@ -57,6 +62,48 @@
     return builder.build();
   }
 
+  private static TestStream<?> fromProto(
+      RunnerApi.TestStreamPayload testStreamPayload, RunnerApi.Components components)
+      throws IOException {
+
+    Coder<Object> coder =
```

Comment:
I kind of want this to be a T even though it doesn't really mean anything

(no action required)